widgetssdk/src/main/java/com/glia/widgets/entrywidget/EntryWidgetView.kt

```diff
@@ -0,0 +1,44 @@
+package com.glia.widgets.entrywidget
+
+import android.content.Context
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import com.glia.widgets.R
+import com.glia.widgets.di.Dependencies
+import com.glia.widgets.entrywidget.adapter.EntryWidgetAdapter
+import com.glia.widgets.entrywidget.adapter.EntryWidgetItemDecoration
+import com.glia.widgets.helper.getDrawableCompat
+
+internal class EntryWidgetView(
+    context: Context
+) : RecyclerView(context, null, 0), EntryWidgetContract.View {
```

Comment:
There is this concern in the project plan:
>If we use RecyclerView for showing the engagement items (Chat, Audio, Video, Messaging), integrators [would have to use](https://armanco.medium.com/how-to-use-vertical-recyclerview-inside-scrollview-in-android-b72337285517) androidx.core.widget.NestedScrollView as a parent view to handle embedded view height change. We cannot limit the integrators. Taking into account that the number of items is final and not that big, we can show them using ConstraintLayout. In this case, embedded view height change will be animated and handled properly.

You mentioned this yesterday, during our sync but I forgot to clarify if the following case will be handled properly (see last step):
- Entry Widget displays two items
- media types availability is updated and one more item becomes available
- will Entry Widget be able to redraw items and occupy more space?

Reply:
Will it help if we disable scroll in RecyclerView?

Reply:
I didn't add animation, but the view should resize with no problem.

https://github.com/user-attachments/assets/79c0887e-e8ae-4529-87f6-fb6b6d77259a

Reply:
but what about the embedded view?

Reply:
@DavDo The bottom sheet includes exactly the embedded view. The only difference is the border, which will be added after the default theme is implemented.

Reply:
Bottom sheet view container adjusts the height, that's good. However, could you please add the embedded view to the MainFragment hierarchy and test the same resizing there?

Reply:
Hm, looks like it's working well. 👍 
We will still need to brainstorm various ideas how integrators can use it and test.